js/bloodmallet_standard.js

```diff
@@ -1251,53 +1235,33 @@ function create_color(dps, min_dps, max_dps) {
     console.log("create_color");
 
   // colour of lowest DPS
-  let color_min = [0, 255, 255];
-  // additional colour step between min and max
-  let color_mid = [255, 255, 0];
```

Comment:
I'm happy with having the option to let the colour flow over a third one. I don't want to miss that.